apps/hosts/urls.py

```diff
@@ -12,5 +12,6 @@
         name='get_challenge_host_list'),
     url(r'challenge_host_team/(?P<challenge_host_team_pk>[0-9]+)/challenge_host/(?P<pk>[0-9]+)$',
         views.challenge_host_detail, name='get_challenge_host_details'),
-
+    url(r'(?P<challenge_host_team_pk>[0-9]+)$',
```

Comment:
I guess you also need to add some other params in url so that the request comes to this. your url can be used to serve get/update/delete for a particular challenge host team. so you will explicitly need to mention something like remove